printer_mods/README.md

```diff
@@ -134,6 +134,8 @@ like so:
 | |[Din Rail mount](./hijax_pl/din_rail_mount) |Adapter to attach DIN Rail to 2020 extrusion. |:x: |:x: |:heavy_check_mark: |:x:|
 | |[Controler Board adapters](./hijax_pl/ender_compatible_controller_mount) |DIN (rotated) and legacy adapters for boards like SKR mini E3. |:grey_question: |:x: |:heavy_check_mark: |:x:|
 |hoschi| [FYSETC Spider DIN Rail Mount](./hoschi/FYSETC_Spider_DIN_Mount) | DIN rail mount for FYSETC Spider | :x: | :grey_question: | :heavy_check_mark: | :grey_question: |
+|hymness1| [Quickdraw_probe_Voron_Switchwire](./hymness1/Quickdraw_probe_Voron_Switchwire) | Detachable magnetic probe with docking system for VSW | :x: | :x: | :x: | :heavy_check_mark: |
```

Comment:
Please add an `:x:` in the Trident column.

Please remove underscores from inside square brackets, that is the text shown to the user:

    [Quickdraw probe Voron Switchwire](./hymness1/Quickdraw_probe_Voron_Switchwire)

Same thing on next line.